content/Guides/DistributedComplianceLedger.md

```diff
@@ -0,0 +1,142 @@
+    - [Web User Interface](#web-user-interface)
+    - [Command Line Interface](#command-line-interface)
+  - [Observer Nodes](#observer-nodes)
+    - [Main-Net public ONs](#main-net-public-ons)
+    - [Test-Net public ONs](#test-net-public-ons)
+- [FAQs](#faqs)
+  - [Why is my Vendor information not listed in the DCL?](#why-is-my-vendor-information-not-listed-in-the-dcl)
+  - [When should I write my product information to the Main-Net DCL?](#when-should-i-write-my-product-information-to-the-main-net-dcl)
+- [Contact](#contact)
+
+## DCL Introduction
+The DCL is a blockchain-based system owned and hosted by Alliance members. It is used by the Matter protocol for storing information such as:
+
+| Information          | Description                                               | Account Role involved                                                                    |
+| -------------------- | --------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
+| Vendor               | Information about the manufacturer or vendor of a device. | Data is entered by an approved Vendor account.                                             |
+| Product              | Information about the product device including description, software version, OTA information, etc. | Data is entered by an approved Vendor account.   |
+| Compliance status    | Information about the certification status including certification ID, certification date, etc. | Data is entered by the Alliance's Certification Center account.|
+| PAA Root CAs         | Information about Product Attestation Authorities (PAA) Root CAs used by commissioners during Device Attestation process. | Data is proposed by the Alliance's Trustee and approved by other DCL Trustees.|
+
+Additional information available in the [Distributed Compliance Ledger (DCL) Policies, Procedure and Governance](https://groups.csa-iot.org/wg/members-all/document/26075).
+
+### Types of nodes that operate in the DCL
+
+| Type of Node         | Description                                                                                                                                          |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Validator Node (VN)  | It participates in creating a "consensus" process to add information into the ledger.                                                                |
+| Sentry Node (SN)     | It doesn\'t participate in consensus and wraps the VN representing it for the rest of the network as one of the ways for DDoS and access protection. |
+| Observer Node (ON)   | It doesn\'t participate in consensus and is optimized for public data read and authenticated write interactions with the ledger.                     |

```

Comment:
Can we briefly define what the "consensus process" is?  Perhaps before the table?  Since it's an important distinction for the nodes.

Also I'd drop the "It" from each description

"Participates in creating a consensus process..."
"Doesn't participate in consensus..."

Reply:
Done